homeassistant/components/asuswrt/router.py

```diff
@@ -424,7 +424,7 @@ async def _get_nvram_info(api: AsusWrt, info_type: str) -> dict[str, Any]:
     info = {}
     try:
         info = await api.async_get_nvram(info_type)
-    except OSError as exc:
+    except (OSError, UnicodeDecodeError) as exc:
```

Comment:
Longer term it would be better if the library caught these errors and raised documented library specific errors instead. How should the library user know to catch `UnicodeDecodeError`?

Reply:
[PR #78](https://github.com/kennedyshead/aioasuswrt/pull/78) is already merged in `aioasuswrt` library. When new library version will be available I will rollback this change.